github/issues_labels.go

```diff
@@ -12,13 +12,14 @@ import (
 
 // Label represents a GitHub label on an Issue
 type Label struct {
+	ID    *int    `json:"id,omitempty"`
 	URL   *string `json:"url,omitempty"`
 	Name  *string `json:"name,omitempty"`
 	Color *string `json:"color,omitempty"`
 }
 
 func (l Label) String() string {
-	return fmt.Sprint(*l.Name)
+	return *l.Name
```

Comment:
Sounds good. I actually dug up more history behind that decision, and filed #595 to deal with resolving it (definitely a good idea to leave it out of this PR).